src/components/logger.ts

```diff
@@ -35,6 +39,65 @@ export class Logger {
         this.compilerLogPanel.clear()
     }
 
+    displayProgress(
+        current: number,
+        knownEndpoint?: number,
+        tooltip: string = ''
+      ) {
+        if (current === 0) {
+          if (knownEndpoint === 0) {
+            this.status2.text = ''
+            this.status2.tooltip = ''
+            return
+          } else {
+            this.status2.text = 'Preamble'
+            this.status2.tooltip = ''
+            return
+          }
+        }
+
+        const generateProgressBar = (proportion: number, length: number) => {
+          const wholeCharacters = Math.trunc(length * proportion)
+          const extraEights = Math.round(
```

Comment:
This should be `extraEighths` (an extra `h` before the `s`). Also applies to the other variables.

Reply:
Applied in 0a24c2c32033d2920b37254642988e5004bf213d